recipe/meta.yaml

```diff
@@ -26,18 +26,18 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - boost-cpp
+    - boost-cpp >=1.74
```

Comment:
Why this change? This will hinder future `boost-cpp` migrations

Reply:
(Isuru always faster :D)

Reply:
From above: `For some reason the Linux build is only for boost-cpp 1.74 while Mac OS X is build for 1.72 and 1.74 - for now I restricted mshr to 1.74 and higher`

Reply:
I'd assume this was only a misinterpretation of the selector which reads "do boost 1.72 and 1.74, unless on ARM Macs, for those only 1.72".